<ui version="4.0" >
 <author>GeometryFactory</author>
 <class>Constrained_Delaunay_triangulation_2</class>
 <widget class="QMainWindow" name="Constrained_Delaunay_triangulation_2" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>CGAL Constrained Delaunay Triangulation</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QGraphicsView" name="graphicsView" >
      <property name="focusPolicy" >
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="verticalScrollBarPolicy" >
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy" >
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="transformationAnchor" >
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="fileToolBar" >
   <property name="windowTitle" >
    <string>File Tools</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionClear" />
   <addaction name="actionLoadConstraints" />
   <addaction name="actionSaveConstraints" />
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>Visualization Tools</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionInsertPolyline" />
   <addaction name="actionShowDelaunay" />
   <addaction name="actionShowVoronoi" />
   <addaction name="actionCircumcenter" />
   <addaction name="actionRecenter" />
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="separator" />
    <addaction name="actionClear" />
    <addaction name="actionLoadConstraints" />
    <addaction name="actionSaveConstraints" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionInsertRandomPoints" />
   </widget>
   <widget class="QMenu" name="menuTools" >
    <property name="title" >
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionInsertPolyline" />
    <addaction name="actionShowDelaunay" />
    <addaction name="actionShowVoronoi" />
    <addaction name="separator" />
    <addaction name="actionRecenter" />
    <addaction name="actionMakeGabrielConform" />
    <addaction name="actionMakeDelaunayConform" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menuTools" />
  </widget>
  <action name="actionAbout" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAboutCGAL" >
   <property name="text" >
    <string>About &amp;CGAL</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionInsertRandomPoints" >
   <property name="text" >
    <string>&amp;Insert random points</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionInsertPolyline" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons/Input.qrc" >:/cgal/Input/inputPolyline.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Insert Polyline</string>
   </property>
   <property name="toolTip" >
    <string>Insert Point or Polyline</string>
   </property>
   <property name="statusTip" >
    <string>Left: Insert vtx | Right: Final vtx | Del: Delete vtx</string>
   </property>
  </action>
  <action name="actionClear" >
   <property name="icon" >
    <iconset resource="../icons/File.qrc" >:/cgal/fileToolbar/fileNew.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Clear</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionShowVoronoi" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons/Triangulation_2.qrc" >:/cgal/Triangulation_2/Voronoi_diagram_2.png</iconset>
   </property>
   <property name="text" >
    <string>Show &amp;Voronoi Diagram</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+V</string>
   </property>
   <property name="visible" >
    <bool>false</bool>
   </property>
  </action>
  <action name="actionShowDelaunay" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons/Triangulation_2.qrc" >:/cgal/Triangulation_2/Delaunay_triangulation_2.png</iconset>
   </property>
   <property name="text" >
    <string>Show &amp;Delaunay Triangulation</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionLoadConstraints" >
   <property name="icon" >
    <iconset resource="../icons/File.qrc" >:/cgal/fileToolbar/fileOpen.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Load Constraints...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSaveConstraints" >
   <property name="icon" >
    <iconset resource="../icons/File.qrc" >:/cgal/fileToolbar/fileSave.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Save Constraints...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCircumcenter" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="Constrained_Delaunay_triangulation_2.qrc" >:/cgal/Actions/icons/circumcenter.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Circumcenter</string>
   </property>
   <property name="toolTip" >
    <string>Draw circumcenter</string>
   </property>
  </action>
  <action name="actionRecenter" >
   <property name="icon" >
    <iconset resource="../icons/Input.qrc" >:/cgal/Input/zoom-best-fit</iconset>
   </property>
   <property name="text" >
    <string>Re&amp;center the viewport</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionMakeGabrielConform" >
   <property name="text" >
    <string>Make Gabriel conform</string>
   </property>
  </action>
  <action name="actionMakeDelaunayConform" >
   <property name="text" >
    <string>Make Delaunay conform</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Constrained_Delaunay_triangulation_2.qrc" />
  <include location="../icons/File.qrc" />
  <include location="../icons/Logos.qrc" />
  <include location="../icons/Triangulation_2.qrc" />
  <include location="../icons/Input.qrc" />
 </resources>
 <connections/>
</ui>
